apps/docs/docs/reference/ui/molecules/Tag.mdx

```diff
@@ -0,0 +1,81 @@
+---
+description: Interactive badges.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Badge___bcedffeb307527c1606be224a7fb5ac1.png
```

Comment:
todo: Update this image later.

Reply:
@carolinamenezes can u please update this image later too? thanks

The [component](https://faststore-git-feat-tag-component-fs-605-faststore.vercel.app/reference/ui/molecules/Tag) looks like this:
Badge with a close button

<img width="177" alt="image" src="https://user-images.githubusercontent.com/3356699/204796988-7394765b-fa69-497a-85a5-3cc26d25f23b.png">